syncano/models/classes.py

```diff
@@ -110,7 +110,7 @@ class Object(Model):
     created_at = fields.DateTimeField(read_only=True, required=False)
     updated_at = fields.DateTimeField(read_only=True, required=False)
 
-    owner = fields.IntegerField(label='owner id', required=False, read_only=True)
+    owner = fields.IntegerField(label='owner id', required=False, read_only=False)
```

Comment:
I need to refactor ```read_only/required``` defaults, because of mess in classes. There are several places where it's redundant kwarg.